--- ./erts/emulator/drivers/common/inet_drv.c.orig	2017-08-14 11:11:37.438436967 +0530
+++ ./erts/emulator/drivers/common/inet_drv.c	2017-08-14 11:11:40.978436182 +0530
@@ -435,6 +435,12 @@
 #     define sctp_adaptation_layer_event sctp_adaption_layer_event
 #endif
 
+#ifdef __NetBSD__
+#undef SCTP_DELAYED_ACK_TIME
+#undef HAVE_DECL_SCTP_DELAYED_ACK_TIME
+#define HAVE_DECL_SCTP_DELAYED_ACK_TIME 0
+#endif
+
 #if defined(__GNUC__) && defined(HAVE_SCTP_BINDX)
 static typeof(sctp_bindx) *p_sctp_bindx = NULL;
 #else
