--- a/erts/etc/unix/run_erl.c	2017-08-14 12:12:03.735641919 +0530
+++ b/erts/etc/unix/run_erl.c	2017-08-14 12:10:42.327517122 +0530
@@ -46,7 +46,7 @@
      /* On OS X and BSD, we must leave _XOPEN_SOURCE undefined in order for
       * the prototype of vsyslog() to be included.
       */
-#    if !(defined(__APPLE__) || defined(__FreeBSD__) || defined(__DragonFly__))
+#    if !(defined(__APPLE__) || defined(__FreeBSD__) || defined(__DragonFly__) || defined(__NetBSD__))
 #      define _XOPEN_SOURCE 600
 #    endif
 #  endif
